client/src/actions/ConfigurationsActions.ts

```diff
@@ -138,6 +174,7 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
     return true;
   }
 
+
```

Comment:
Have you checked this PR for lint errors?
I think that double empty rows will appear in the linter as warning.